client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -272,7 +269,7 @@ func (d *DockerOps) StartSMBServer(volName string) (int, string, bool) {
 	}
 
 	// Wait till service container starts
-	ticker := time.NewTicker(checkDuration)
+	ticker := time.NewTicker(checkTicker)
```

Comment:
Why we change the duration between successive checks?

Reply:
5 seconds is too long.
Especially every first time the plugin always needs to wait for the first ticker to finish, means it always take 5 seconds before check if the server is up. However when I tested, docker can start the service quite fast when the image is already loaded. So I reduced the ticker here to 1 second to reduce the delay of the first mount function.